src/rgw/rgw_admin.cc

```diff
@@ -2690,10 +2690,6 @@ static int trim_sync_error_log(int shard_id, const ceph::real_time& start_time,
   // unreachable
 }
 
-JSONFormattable& get_tier_config(RGWRados *store) {
```

Comment:
This function is a lot more interesting if you assume the German meaning of 'tier'.